### ISSUE TITLE
8 0.33 ^ → 1

### ISSUE THREAD

**Issue by cab404** [Vifon/GravCalc#2]
or basically powering to fractions done wrong

**Comment by vifon** [comment]
Yeah, I'm fully aware of it. See [this comment](https://github.com/Vifon/GravCalc/blob/5d175c79a21bea73384cbe3e1f8450c58457cd39/src/fixed.c#L251). I didn't have the strength to implement roots by hand. All the "fractional" math is implemented manually on fixed point numbers due to the very limited support for floating points on Pebble (Cortex-M3) and more importantly its standard library (no printf support).

I'm leaving this ticket open because, well, it's indeed the current state of things. I hope to close it someday. ;)
